### PR TITLE
Expose volcano components (controller, scheduler, etc.) log level control to the helm chat values

### PR DESCRIPTION
**Pull request by chenshiwei-io**
Expose volcano components (controller, scheduler, etc.) log level control to the helm chat values
#3619

### CODE REVIEW

**Comment by chenshiwei-io** [comment]
hello, please help me review the pr. thanks

**Review by hwdef — COMMENTED** [review]
/ok-to-test
/lgtm

Thanks for your contributions!

File: installer/helm/chart/volcano/values.yaml [hunk 1]
```diff
@@ -118,8 +118,11 @@ custom:
 #      cpu: 300m
 #      memory: 300Mi
   admission_resources: ~
+  admission_log_level: 4
```
Comment (Monokaix) [inline installer/helm/chart/volcano/values.yaml:121]: Maybe put these log reltaed together and append to the end of the file is better.

Comment (chenshiwei-io) [reply installer/helm/chart/volcano/values.yaml]: ok, It's done

**Review by hwdef — COMMENTED** [review]
/lgtm

**Comment by Monokaix** [comment]
Hi, you can use git commit --ame --no-edit -s and for push again to retest.

**Comment by chenshiwei-io** [comment]
> Hi, you can use git commit --ame --no-edit -s and for push again to retest.

It's done

**Comment by Monokaix** [comment]
Please squash to one commit.

**Comment by chenshiwei-io** [comment]
> Please squash to one commit.

I got it done.

**Review by william-wang — APPROVED** [review]
/approve

**Comment by chenshiwei-io** [comment]
thanks

**Pull request merged**